apps/ledger-live-mobile/e2e/page/trade/receive.page.ts

```diff
@@ -38,22 +38,34 @@ export default class ReceivePage {
   step2Accounts = () => getElementById("receive-header-step2-accounts");
   step2Networks = () => getElementById("receive-header-step2-networks");
 
-  openViaDeeplink() {
-    return openDeeplink(baseLink);
+  async openViaDeeplink() {
+    await openDeeplink(baseLink);
   }
 
   async receiveViaDeeplink(currencyLong?: string) {
     const link = currencyLong ? baseLink + currencyParam + currencyLong : baseLink;
     await openDeeplink(link);
   }
 
+  async expectFirstStep() {
+    await expect(this.step1HeaderTitle()).toBeVisible();
+  }
+
+  async expectSecondStep(networks: string[]) {
+    await expect(this.step2HeaderTitle()).toBeVisible();
+    await expect(this.step2Networks()).toBeVisible();
+    for (const network of networks) {
+      await expect(getElementById(this.currencyNameId(network))).toBeVisible();
+    }
+  }
+
   async selectCurrencyRow(currencyId: string) {
```

Comment:
this method is not in use anywhere, do we need to keep it anyway?

Reply:
Deleted